assets/json-schema-faker.js

```diff
@@ -24547,7 +24549,12 @@ function extend() {
           return;
       }
       if (optionAPI('useExamplesValue') && 'example' in schema) {
-        return schema.example;
+        var result = validateSchema(schema, schema.example);
+
+        // Use example only if valid 
+        if (result && result.length === 0) {
```

Comment:
@VShingala if this is false, are we guaranteed to have a fallback?

Reply:
Yeah, it will not use `schema.example` anymore, and example will be faked according to changes done for using faked value instead schema. The flow is similar to not having example at all.